suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -1153,3 +1154,52 @@ export const isTokenMatchesSearch = (token: TokenInfo, search: string) => {
         token.policyId?.toLowerCase().includes(search)
     );
 };
+
+export const isDebugOnlyAccountType = (
```

Comment:
tbh this file is huge, I would put those new methods into a new file and same with tests. It is going to be better also because the fact that you create helper methods and types for the tests

Reply:
I extracted it into a new file

Reply:
Where?

Reply:
Oh sorry, I only extracted tests, will extract this as well🤦🏻‍♀️